src/components/PlaylistModal/PlaylistModal.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { useUserLists } from "../../context";
+import { useToggle } from "../../hooks";
+import {
+  addNewPlaylist,
+  addVideoInPlaylist,
+  removeVideoFromPlaylist,
+} from "../../services";
+import { isVideoExistsInList } from "../../utils";
+import "./PlaylistModal.css";
+
+function PlaylistModal({ setIsModalVisible, video }) {
```

Comment:
Try to provide some default props incase the value is not passed then It will not break since it has default values.
Explore `defaultProps`

Reply:
Yes, nice idea. i will make the changes.